inst/private/python_header.py

```diff
@@ -11,16 +11,12 @@
 sys.ps1 = ""; sys.ps2 = ""
 
 
-def myerr(e):
-    # hardcoded in case no xml
-    print("<output_block>")
-    print("<item>\n<f>9999</f>\n<f>")
-    print(str(e[0]).replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;"))
-    print("</f><f>")
-    print(str(e[1]).replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;"))
-    print("</f>\n</item>")
-    print("</output_block>\n")
-
+def myerr(error):
+    try:
+        octoutput_drv(("INTERNAL_PYTHON_ERROR", error))
+    except:
+        pass
+    raise
```

Comment:
Do we *need* to raise if the `octoutput_drv` works?  Or maybe its safer to do so?

(If I understand correctly, raise will just print the stacktrace to stderr)